### PR TITLE
Add LXGW series fonts

### LINKED ISSUE #272: add LxgwNeoZhiSong
Hello, I want to suggest a new font.

> homepage: https://github.com/lxgw/LxgwNeoZhiSong

Actually this repositery cotains 2 fonts: LXGWHeartSerifCHS and LXGWNeoZhiSongCHS, and LXGWHeartSerifCHS is a modified version of LXGWNeoZhiSongCHS.

### PR DESCRIPTION
**Pull request by NightFurySL2001**
* Update LXGW Wenkai with breaking v1.500 changes
* Add LXGW NeoZhiSong (fix #272 )

### CODE REVIEW

File: bucket/LXGWWenKaiGB.json [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        "script": [
+            "$fontInstallDir = if ($global) { \"$env:windir\\Fonts\" } else { \"$env:LOCALAPPDATA\\Microsoft\\Windows\\Fonts\" }",
+            "$registryRoot = if ($global) { \"HKLM\" } else { \"HKCU\" }",
+            "$registryKey = \"${registryRoot}:\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Fonts\"",
+            "Get-ChildItem $dir -Filter '*.ttf' | ForEach-Object {",
+            "    Remove-ItemProperty -Path $registryKey -Name $_.Name.Replace($_.Extension, ' (TrueType)') -Force -ErrorAction SilentlyContinue",
+            "    Remove-Item \"$fontInstallDir\\$($_.Name)\" -Force -ErrorAction SilentlyContinue",
+            "}",
+            "if ($cmd -eq \"uninstall\") {",
+            "    Write-Host \"Font family 'LXGWWenKai' has been uninstalled and will not be present after restarting your computer.\" -Foreground Magenta",
```
Comment (jonz94) [inline bucket/LXGWWenKaiGB.json:92]:
```suggestion
            "    Write-Host \"Font family '$app' has been uninstalled and will not be present after restarting your computer.\" -Foreground Magenta",
```

We can simply use `'$app'` for the font name.

File: bucket/LXGWWenKaiMonoGB.json [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        "script": [
+            "$fontInstallDir = if ($global) { \"$env:windir\\Fonts\" } else { \"$env:LOCALAPPDATA\\Microsoft\\Windows\\Fonts\" }",
+            "$registryRoot = if ($global) { \"HKLM\" } else { \"HKCU\" }",
+            "$registryKey = \"${registryRoot}:\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Fonts\"",
+            "Get-ChildItem $dir -Filter '*.ttf' | ForEach-Object {",
+            "    Remove-ItemProperty -Path $registryKey -Name $_.Name.Replace($_.Extension, ' (TrueType)') -Force -ErrorAction SilentlyContinue",
+            "    Remove-Item \"$fontInstallDir\\$($_.Name)\" -Force -ErrorAction SilentlyContinue",
+            "}",
+            "if ($cmd -eq \"uninstall\") {",
+            "    Write-Host \"Font family 'LXGWWenKai' has been uninstalled and will not be present after restarting your computer.\" -Foreground Magenta",
```
Comment (jonz94) [inline bucket/LXGWWenKaiMonoGB.json:92]:
```suggestion
            "    Write-Host \"Font family '$app' has been uninstalled and will not be present after restarting your computer.\" -Foreground Magenta",
```

File: bucket/LXGWWenKaiMonoTC.json [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        "script": [
+            "$fontInstallDir = if ($global) { \"$env:windir\\Fonts\" } else { \"$env:LOCALAPPDATA\\Microsoft\\Windows\\Fonts\" }",
+            "$registryRoot = if ($global) { \"HKLM\" } else { \"HKCU\" }",
+            "$registryKey = \"${registryRoot}:\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Fonts\"",
+            "Get-ChildItem $dir -Filter '*.ttf' | ForEach-Object {",
+            "    Remove-ItemProperty -Path $registryKey -Name $_.Name.Replace($_.Extension, ' (TrueType)') -Force -ErrorAction SilentlyContinue",
+            "    Remove-Item \"$fontInstallDir\\$($_.Name)\" -Force -ErrorAction SilentlyContinue",
+            "}",
+            "if ($cmd -eq \"uninstall\") {",
+            "    Write-Host \"Font family 'LXGWWenKai' has been uninstalled and will not be present after restarting your computer.\" -Foreground Magenta",
```
Comment (jonz94) [inline bucket/LXGWWenKaiMonoTC.json:92]:
```suggestion
            "    Write-Host \"Font family '$app' has been uninstalled and will not be present after restarting your computer.\" -Foreground Magenta",
```

File: bucket/LXGWWenKaiTC.json [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+        "script": [
+            "$fontInstallDir = if ($global) { \"$env:windir\\Fonts\" } else { \"$env:LOCALAPPDATA\\Microsoft\\Windows\\Fonts\" }",
+            "$registryRoot = if ($global) { \"HKLM\" } else { \"HKCU\" }",
+            "$registryKey = \"${registryRoot}:\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Fonts\"",
+            "Get-ChildItem $dir -Filter '*.ttf' | ForEach-Object {",
+            "    Remove-ItemProperty -Path $registryKey -Name $_.Name.Replace($_.Extension, ' (TrueType)') -Force -ErrorAction SilentlyContinue",
+            "    Remove-Item \"$fontInstallDir\\$($_.Name)\" -Force -ErrorAction SilentlyContinue",
+            "}",
+            "if ($cmd -eq \"uninstall\") {",
+            "    Write-Host \"Font family 'LXGWWenKai' has been uninstalled and will not be present after restarting your computer.\" -Foreground Magenta",
```
Comment (jonz94) [inline bucket/LXGWWenKaiTC.json:92]:
```suggestion
            "    Write-Host \"Font family '$app' has been uninstalled and will not be present after restarting your computer.\" -Foreground Magenta",
```

File: bucket/LXGWWenKaiMono.json [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
-    "homepage": "https://github.com/lxgw/LxgwWenKai",
+    "homepage": "https://github.com/lxgw/LxgwWenkai",
     "license": "OFL-1.1",
     "url": [
-        "https://github.com/lxgw/LxgwWenKai/releases/download/v1.330/LXGWWenKaiMono-Bold.ttf",
-        "https://github.com/lxgw/LxgwWenKai/releases/download/v1.330/LXGWWenKaiMono-Light.ttf",
-        "https://github.com/lxgw/LxgwWenKai/releases/download/v1.330/LXGWWenKaiMono-Regular.ttf"
+        "https://github.com/lxgw/LxgwWenkai/releases/download/v1.501/LXGWWenKaiMono-Light.ttf",
+        "https://github.com/lxgw/LxgwWenkai/releases/download/v1.501/LXGWWenKaiMono-Regular.ttf",
+        "https://github.com/lxgw/LxgwWenkai/releases/download/v1.501/LXGWWenKaiMono-Medium.ttf"
```
Comment (jonz94) [inline bucket/LXGWWenKaiMono.json:9]:
```suggestion
        "https://github.com/lxgw/LxgwWenKai/releases/download/v1.501/LXGWWenKaiMono-Light.ttf",
        "https://github.com/lxgw/LxgwWenKai/releases/download/v1.501/LXGWWenKaiMono-Regular.ttf",
        "https://github.com/lxgw/LxgwWenKai/releases/download/v1.501/LXGWWenKaiMono-Medium.ttf"
```

**Review by jonz94 — CHANGES_REQUESTED** [review]
Thanks for the PR!

Before merging, I'd like to suggest a few changes.
Feel free to reach out if you have any questions 🤗

**Comment by NightFurySL2001** [comment]
All fixed

**Review by jonz94 — APPROVED** [review]

**Comment by jonz94** [comment]
LGTM! Thanks for your contribution.

**Pull request merged**